### PR TITLE
Feature/us300 atrium screen list

### PR DESCRIPTION
**Pull request by IngageKrizJ**
Changes for atrium display, includes working scrolling using jQuery Cycle2

### CODE REVIEW

File: crossroads.net/app/events/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+							evtTime.append(evtMeridian);
+							evtName.append(evts[i].name);
+							evtLocation.append(evts[i].location);
+							
+							row.append(evtTime);
+							row.append(evtName);
+							row.append(evtLocation);
+							
+							tbody.append(row);
+						}
```
Comment (silbermm) [inline crossroads.net/app/events/index.js]: I think that we should be consistent with our Jquery naming: i.e '$' or 'jQuery', but not both

File: crossroads.net/app/events/index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+						var atriumTable = $('<table class="table atrium-table">');
+						atriumTable.append(tbody);
+						atriumCycleCell.append(atriumTable);
+						atriumCycleCell.append($('<hr class="atrium-border" />'));
+						element.append(atriumCycleCell);
+						
+						var marqueeElement = jQuery(".atrium-body");
+						var marqueeHeight = jQuery(marqueeElement).height();
+						var windowHeight = jQuery(window).height();
+						$log.debug("Marquee Height: " + marqueeHeight);
```
Comment (silbermm) [inline crossroads.net/app/events/index.js]: Remove $log.debugs before merging into development

**Comment by silbermm** [comment]
There is an extra directory called crds_angular that needs to be removed before we can merge

File: crossroads.net/app/events/index.js (hunk 1 above)
Comment (IngageKrizJ) [inline crossroads.net/app/events/index.js]: Updated, changed to use $ notation everywhere in directive.

**Comment by silbermm** [comment]
:+1:

**Pull request merged**